arrows/matlab/mxarray.h

```diff
@@ -831,247 +773,208 @@ class MxArray
-  /** Container with fundamental numerics, i.e. vector<double>.
-   */
+  /// Container with fundamental numerics, i.e. vector<double>.
   template < typename Container >
   static mxArray* fromInternal( const typename std::enable_if<
                                   MxArithmeticCompound< Container >::value,
                                   Container >::type& value );
-  /** Container with complex numbers, i.e. vector<complex<double>>.
-   */
+  /// Container with complex numbers, i.e. vector<complex<double>>.
   template < typename Container >
   static mxArray* fromInternal( const typename std::enable_if<
                                   MxComplexCompound< Container >::value, Container >::type& value );
-  /** Char type */
+  /// Char type
   template < typename T >
   static mxArray* fromInternal( const typename std::enable_if<
                                   MxCharType< T >::value, T >::type& value );
-  /** Containter with signed char.
-   */
+  /// Containter with signed char.
   template < typename Container >
   static mxArray* fromInternal( const typename std::enable_if<
                                   ( MxCharCompound< Container >::value ) &&
                                   ( std::is_signed< typename Container::value_type >::value ),
                                   Container >::type& value );
-  /** Container with unsigned char.
-   */
+  /// Container with unsigned char.
   template < typename Container >
   static mxArray* fromInternal( const typename std::enable_if<
                                   ( MxCharCompound< Container >::value ) &&
                                   ! ( std::is_signed< typename Container::value_type >::value ),
                                   Container >::type& value );
-  /** Logicals.
-   */
+  /// Logicals.
   template < typename T >
   static mxArray* fromInternal( const typename std::enable_if<
                                   MxLogicalType< T >::value, T >::type& value );
-  /** Container with logicals.
-   */
+  /// Container with logicals.
   template < typename Container >
   static mxArray* fromInternal( const typename std::enable_if<
                                   MxLogicalCompound< Container >::value, Container >::type& value );
-  /** Container with cell type content.
-   */
+  /// Container with cell type content.
   template < typename Container >
   static mxArray* fromInternal( const typename std::enable_if<
                                   MxCellCompound< Container >::value, Container >::type& value );
 
-  /*************************************************************/
-  /**             Templated mxArray exporters                 **/
-  /*************************************************************/
+  //*************************************************************
+  ///
+  ///***          Templated mxArray exporters                 ***
+  ///
+  //*************************************************************
+
```

Comment:
Likewise. Also, the added (and thus double) blank line is against the style guidelines.